admin/sql/updates/20220322-mbs-12256-mirror.sql

```diff
@@ -0,0 +1,168 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+UPDATE artist_meta
+   SET rating = agg.rating
+  FROM (
+      SELECT artist,
+             round(sum(rating)::REAL /
+                   count(rating)::REAL)::SMALLINT AS rating
```

Comment:
Shouldn't this also use `trunc()` then?

Reply:
Looks like I changed the implementation at some point and didn't recompile the scripts.  Thanks, will fix.